network-api/networkapi/wagtailpages/templates/wagtailpages/blocks/blog_cta_card_with_text_block.html

```diff
@@ -0,0 +1,14 @@
+{% extends "./base_streamfield_block.html" %}
+{% load l10n wagtailcore_tags wagtailimages_tags %}
+
+{% block block_content %}
+    <div class="blog-cta-card-with-text">
+        <div class="blog-cta-card-with-text__card-container blog-cta-card-with-text__card-container--{{ value.alignment }}">
```

Comment:
Hi @jhonatan-lopes , we don't use BEM naming convention in this codebase so I would appreciate if we don't  introduce a new naming pattern. To be honest, there's no one consistent naming convention that we have been sticking to in this codebase 😓 , however, we try to use shorter names possible.

I see that we are also adding new SCSS code in this PR. As we are trying to move away from SCSS, could we convert the rules into Tailwind classes instead? If the rules will be used more than just this block, we can even create custom component on Tailwind config. What do you think? If this sounds good then that also mitigate the naming convention headache here.

Reply:
Hi @mmmavis, thanks for raising these points! It was definitely inertia from working with BEM before :smile: I've migrated the styles to only use Tailwind now.